com.coinspace.wallet.json

```
@@ -0,0 +1,37 @@
+{
+  "id" : "com.coinspace.wallet",
+  "branch" : "stable",
+  "runtime" : "org.freedesktop.Platform",
+  "runtime-version" : "24.08",
+  "runtime-commit" : "6ad4305d3f60009c2a75a59f99c652ab271ea89347e10201e74730cb1a6c0ecd",
+  "sdk" : "org.freedesktop.Sdk",
+  "sdk-commit" : "b7beb7e6d0ee2b1c4c4ec90c1380b569d10b12f05a097aa38072b8fdc148385f",
```

```suggestion
  "sdk" : "org.freedesktop.Sdk",
```

Not sure which bit of the documentation told to put these but it's definitely wrong